moduleroot/Rakefile

```diff
@@ -6,4 +6,4 @@ PuppetLint.configuration.fail_on_warnings = true
 <% checks.each do |check| -%>
 PuppetLint.configuration.send('<%= check %>')
 <% end -%>
-PuppetLint.configuration.ignore_paths = ["spec/**/*.pp", "pkg/**/*.pp"]
```

Comment:
Until puppet lint 2.0 the line in the Rakefile was ignored anyways, as far as I can tell.